lib/text.rb

```diff
@@ -0,0 +1,23 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class Text
+
+	def initialize (text = Text.new)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)